README.md

```diff
@@ -7,7 +7,13 @@
 
 # Overview
 
-This solution pack helps community users to submit their custom solution to FSR team.
+The **Submit Your Content** solution pack empowers community members to seamlessly submit their solutions for review on FortiSOAR’s Content Hub. By launching their FortiSOAR instance and leveraging playbooks, users can streamline the submission process, ensuring efficient handling of their valuable contributions.
+
+This solution pack simplifies content submission by breaking down the entire process into manageable steps addressed through user-friendly interfaces.
+
+- **Integrated with FortiSOAR**: Content submission is seamlessly integrated with FortiSOAR. You can use the [FortiSOAR RDK](https://docs.fortinet.com/fortisoar/connectors/rapiddevkit) to create solution packs, widgets, or connectors directly from the FortiSOAR interface. Once you've created your content, use the **Submit Your Content** solution pack to submit it to the FortiSOAR Integrations team for review and subsequent upload to the Content Hub.
```

Comment:
to create solution packs, widgets, or connectors directly from the FortiSOAR interface ---> we don't create widget and sp using rdk. Also, rdk is not used from the FSR interface.